pep_sphinx_extensions/pep_processor/parsing/pep_banner_directive.py

```diff
@@ -103,3 +102,26 @@ class CanonicalPyPASpecBanner(PEPBanner):
     admonition_class = nodes.attention
 
     css_classes = ["canonical-pypa-spec", "sticky-banner"]
+
+
+class CanonicalTypingSpecBanner(PEPBanner):
+    """Insert a specialized admonition for the typing specification."""
+
+    admonition_pre_template = (
+        "This PEP is a historical document. "
+        "The up-to-date, canonical spec, {link_content}, is maintained on "
+        f"the `typing specs site <{TYPING_SPEC_BASE_URL}>`__."
+    )
+    admonition_pre_text = (
+        "This PEP is a historical document. "
+        "The up-to-date, canonical specifications are maintained on "
+        f"the `typing specs site <{TYPING_SPEC_BASE_URL}>`__."
+    )
+    admonition_post_text = (
+        "See the `typing specification update process "
+        "<https://typing.readthedocs.io/en/latest/spec/TODO.html>`__ "
+        "for how to propose changes."
```

Comment:
What URL do we want here? Or some other text, or no text?

Reply:
Following https://github.com/python/typing/pull/1573, updated to point to https://typing.readthedocs.io/en/latest/spec/meta.html